EMS/Enquiry/admin.py

```diff
@@ -0,0 +1,4 @@
+from django.contrib import admin
+from .models import Enquiry
+# Register your models here.
+admin.site.register(Enquiry)
```

Comment:
Here 2 lines break should be present and then only you should start your implementations
```suggestion
from django.contrib import admin
from .models import Enquiry


admin.site.register(Enquiry)
```